### ISSUE TITLE
Feature Request: import recipes from App "Recipe Keeper Pro"

### ISSUE THREAD

**Issue by newtogithub-mike** [grocy/grocy#2263]
I was on a long search for an app that could house my various sources of recipes. Books, Magazines, Pdf files, websites.

The most convenient I found was Recipe Keeper Pro and I still love it. Sit on the couch, browse through a precious cook book - find an interesting recipe, take a picture and then magically the app recognizes the letters and mostly also puts it in the right box (like ingedients, how to make it, etc.)
What makes me love this app is the ease of growing this colllection.

So to not only collect recipes but also integrate them into my cooking I would love to import them to grocy. I mean it would be no problem to just import them and then link the ingredients to the various items on grocy.

So far I have only found import from websites to grocy.

thanks a lot
Michael

**Comment by berrnd** [comment]
> So far I have only found import from websites to grocy.

Can't be within Grocy, since it doesn't support that - [recipe-buddy](https://github.com/georgegebbett/recipe-buddy) can do that (I guess, don't use that personally).

&nbsp;
> I mean it would be no problem to just import them and then link the ingredients to the various items on grocy.

I mean, that's the technically challenging part. Each and every ingredient, each and every quantity unit and everything else need to be mapped to the corresponding object in Grocy, providing a great UI to make it easily possible to match that all and to create missing objects right there on the fly (or something like that). If that all sounds like "no problem", feel free to start that right away.

&nbsp;
In conclusion a duplicate of #210 - please check existing requests before opening new ones. For this project, in any case something like that would be build in a generic way. Specific app integrations are something for separate projects.
